metaserver/src/main/scala/scala/meta/languageserver/Semanticdbs.scala

```diff
@@ -14,14 +16,15 @@ object Semanticdbs extends LazyLogging {
   def loadFromTextDocument(
     compiler: Global,
     td: VersionedTextDocumentIdentifier,
-    content: String
-  ): scala.meta.semanticdb.Database = {
+    content: String,
+    cwd: AbsolutePath
+  ): Database = {
     val documents = try {
       List(InteractiveSemanticdb.toDocument(compiler, content, Uri.toPath(td.uri).get.toLanguageServerUri, 10000))
     } catch {
       case e: ParseException => Nil
     }
-    scala.meta.semanticdb.Database(documents)
+    semanticdb.Database(documents).toSchema(cwd)
```

Comment:
Hmm, I'm not really sure we should do `.toSchema` here since we do `.toDatabase` to feed this back to scalafix later. I propose we call `.toSchema` right before feeding this to the indexer.

Reply:
As discussed offline, I will update this later today!